http-get-product-by-id/index.ts

```diff
@@ -0,0 +1,20 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import productsModel from '../model/product'
+import sctocksModel from '../model/stock'
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+    context.log(req);

```

Comment:
no a blocker. But probably it's better to highlight that function invocation started.